cms/src/main/java/uk/nhs/digital/apispecs/swagger/IsAnyTrueHelper.java

```diff
@@ -0,0 +1,38 @@
+package uk.nhs.digital.apispecs.swagger;
+
+import com.github.jknack.handlebars.Helper;
+import com.github.jknack.handlebars.Options;
+import org.apache.commons.lang.Validate;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+public class IsAnyTrueHelper implements Helper<String> {
+
+    public static final IsAnyTrueHelper INSTANCE = new IsAnyTrueHelper();
+
+    public static final String NAME = "isAnyTrue";
+
+    @Override public Object apply(final String items, final Options options) throws IOException {
+
+        Validate.notEmpty(options.params,"At least one parameter is required but none was provided.");
+        Validate.noNullElements(options.params,"At least one parameter is required but none was provided.");
```

Comment:
noNullElements check is redundant now.

Validate.notEmpty() checks for null

Reply:
As discussed here and on Slack, I don't agree that `noNullElements` is redundant, because it validates a different aspect of the inputs than `notEmpty` does.

Indeed, both `noNullElements` and `notEmpty` check for `options.params` _itself_ being `null` but that's not the point here. The point is that _only_ `noNullElements` explicitly checks whether any of the collection's _elements_ is `null` - as explained above, and on Slack, and as communicated by the method's names themselves.

As it is, when `options.params` has any of its elements `null`, `notEmpty` will pass (correctly), but the fact that there is a `null` element will be ignored without any clear message, and the validation will 'fall through' to the `allElementesOfType`. Luckily, this one will fail but with a message that will mislead into thinking that the failure was because not all elements were boolean, unnecessarily making any potential bug hunting a little harder.

In my view leaving the `noNullElements` validation off is unjustified because keeping it:
a) costs nothing (single, easy to read line of code - which was already there and is now actively being removed),
b) provides a chance to communicate what exactly has failed and why, through an explicit and accurate message that saves the reader time and effort of analysing code in order to find the cause,
c) does not incur any effort of writing test worth mentioning, because adding a test case to verify this validation is trivially simple.

Having said that, this is Monika's explicit decision made after hearing my arguments, therefore I accept this implementation and am resolving this discussion.